docs/symlinks.rst

```diff
@@ -46,11 +46,11 @@ So we simply serve them the documentation:
           This will cause things to hit the Python backend,
           so that proper action can be taken.
 
-CNAMEs
-------
+Custom domains
+--------------
 
-CNAMEs add a bit of difficulty,
-because at the nginx layer we don't know what documentation to serve.
+Custom domains add a bit of difficulty,
+because at the Nginx layer we don't know what documentation to serve.
```

Comment:
I think only the company is capitalized. On the [nginx website](https://nginx.org/), it is consistently lower-cased.